test_security/CMakeLists.txt

```diff
@@ -331,13 +331,17 @@ if(BUILD_TESTING)
     ament_find_gtest()
 
     set(KEYSTORE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/test/test_security_files")
-    set(VALID_ROS_SECURITY_ROOT_DIRECTORY "${KEYSTORE_DIRECTORY}/contexts")
+    if(WIN32)
+      string(REPLACE "/" "\\\\" KEYSTORE_DIRECTORY_NATIVE_PATH "${KEYSTORE_DIRECTORY}")
```

Comment:
Nit^10: this could be done in place removing the need for a new variable and an else cause here

Reply:
I'm not sure if this https://github.com/ros2/system_tests/blob/0ec293e4bd6efc7bf935b9e2885a6bac35856e76/test_security/CMakeLists.txt#L355

will continue working well, as it's a mixture of a cmake and a "native" path.

I'm going to left this as-is, as I don't want to come back to Windows to test.